clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/view/EcrImageProviderSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnaker.clouddriver.ecs.view
+
+import com.netflix.spinnaker.clouddriver.ecs.provider.view.EcrImageProvider
+import org.junit.Test
+import spock.lang.Specification
+
+class EcrImageProviderSpec extends Specification {
+
+  @Test
+  void shouldHandleEcrRepositoryUrl() {
+    given:
+    EcrImageProvider ecrImageProvider = new EcrImageProvider(null, null)
+    String ecrRepositoryUrl = "769716316905.dkr.ecr.us-west-2.amazonaws.com/continuous-delivery:latest"
```

Comment:
If this is your real AWS account ID, you should definitely scrub it before submitting a PR - it's sensitive information.